stdlib/testing/testdata/cov.flux

```diff
@@ -32,23 +35,23 @@ outData = "
 ,,0,CPU,8
 ,,1,RAM,-1.8333333333333333
 "
-
 t_cov = () => {
-    left = testing.loadStorage(csv: inData)
-        |> range(start:2018-05-22T19:53:00Z, stop:2018-05-22T19:55:00Z)
-        |> drop(columns: ["_start", "_stop"])
-        |> filter(fn: (r) => r.user == "user1")
-        |> group(columns: ["_measurement"])
-
-    right = testing.loadStorage(csv: inData)
-        |> range(start:2018-05-22T19:53:00Z, stop:2018-05-22T19:55:00Z)
-        |> drop(columns: ["_start", "_stop"])
-        |> filter(fn: (r) => r.user == "user2")
-        |> group(columns: ["_measurement"])
+	left = testing.loadStorage(csv: inData)
+		|> range(start: 2018-05-22T19:53:00Z, stop: 2018-05-22T19:55:00Z)
+		|> drop(columns: ["_start", "_stop"])
+		|> filter(fn: (r) =>
+			(r.user == "user1"))
+		|> group(columns: ["_measurement"])
+	right = testing.loadStorage(csv: inData)
+		|> range(start: 2018-05-22T19:53:00Z, stop: 2018-05-22T19:55:00Z)
+		|> drop(columns: ["_start", "_stop"])
+		|> filter(fn: (r) =>
+			(r.user == "user2"))
+		|> group(columns: ["_measurement"])
+	got = cov(x: left, y: right, on: ["_time", "_measurement"])
+	want = testing.loadStorage(csv: outData)
 
-    got = cov(x:left, y:right, on: ["_time", "_measurement"])
-    want = testing.loadStorage(csv: outData)
-    return testing.assertEquals(name: "cov", want: want, got: got)
+	return testing.assertEquals(name: "cov", want: want, got: got)
```

Comment:
we can probably upgrade this to the new framework too.  the old one didn't play well with multiple sources.